yarn.lock

```diff
@@ -6937,6 +6937,11 @@ cypress-mochawesome-reporter@3.8.2:
     mochawesome-merge "^4.2.1"
     mochawesome-report-generator "^6.2.0"
 
+cypress-real-events@^1.14.0:
```

Comment:
A version should be pinned. Carets `^` tend to make dependencies less maintainable.

Reply:
done